api/features/views.py

```diff
@@ -752,3 +769,94 @@ def create_segment_override(
     serializer.is_valid(raise_exception=True)
     serializer.save(environment=environment, feature=feature)
     return Response(serializer.data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    request_body=CreateFeatureExportSerializer(),
+    responses={201: CreateFeatureExportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([CreateFeatureExportPermissions])
+def create_feature_export(request: Request) -> Response:
+    serializer = CreateFeatureExportSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+
+    export_features_for_environment.delay(
+        kwargs={
+            "environment_id": serializer.validated_data["environment_id"],
+            "tag_ids": serializer.validated_data["tag_ids"],
+        }
+    )
+
+    return Response(serializer.validated_data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    responses={201: FeatureImportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([FeatureImportPermissions])
+def feature_import(request: Request, environment_id: int) -> Response:
+    uploaded_file = request.FILES["file"]
+
+    if uploaded_file.size > MAX_FEATURE_IMPORT_SIZE:
+        return Response(
+            {"detail": "File is too large"},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+
+    strategy = request.query_params.get("strategy").upper()
+
+    if not strategy:
+        return Response(
+            {"detail": "A feature import strategy query param must be supplied."},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+
+    file_content = uploaded_file.read().decode("utf-8")
```

Comment:
Do we want to do any validation of the file content (maybe a second PR)?

Reply:
Resolved offline: First pass no validation.